core/src/plugins/access.swift/class.swiftAccessDriver.php

```diff
@@ -125,4 +155,10 @@ public function makeSharedRepositoryOptions($httpVars, $repository)
         return $newOptions;
     }
 
+    public function directoryUsage($directoryPath, $repositoryResolvedOptions)
```

Comment:
Can you add a note that this implements only the top-level bucket bytesize, without taking into account the directoryPath?